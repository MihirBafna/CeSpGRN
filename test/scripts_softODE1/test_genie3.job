#!/bin/bash
#SBATCH -J test_genie3
#SBATCH -N 1
#SBATCH -p RM-shared
#SBATCH --ntasks-per-node=4
#SBATCH -t 2:00:00
#SBATCH -o test_genie3.out 
#SBATCH -e test_genie3.err
#SBATCH --mail-user=zzhang834@gatech.edu
#SBATCH --mail-type=ALL

# type 'man sbatch' for more information and options
# this job will ask for 1 full RM node (8 cores) for 5 hours
# this job would potentially charge 640 RM SUs

module load anaconda3
conda activate myenv

# move to working directory
# this job assumes:
# - all input data is stored in this directory
# - all output should be stored in this directory
# - please note that groupname should be replaced by your groupname
# - username should be replaced by your username
# - path-to-directory should be replaced by the path to your directory where the executable is

cd /ocean/projects/cis210014p/ziqi/DynGRN/test/scripts_softODE1

# run pre-compiled program which is already in your project space
python test_genie3.py