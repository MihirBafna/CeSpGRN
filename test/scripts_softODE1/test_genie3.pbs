#PBS -N test_genie3
#PBS -q hive-himem
#PBS -l nodes=1:ppn=4
#PBS -l walltime=36:20:00
#PBS -j oe
#PBS -o test_genie3.out
#PBS -m abe                            
#PBS -M zzhang834@gatech.edu  

cd $PBS_O_WORKDIR
module load anaconda3
conda activate myenv
python test_genie3.py