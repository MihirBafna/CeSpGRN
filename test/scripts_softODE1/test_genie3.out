test results:
ngenes_20_interval_50_stepsize_0.0001

model: random
mean kendall-tau: -0.01915247940617325
mean pearson: -0.02632861878851484
mean cosine similarity: -0.03554611841537868


model: genie_standardscaler
mean kendall-tau: -0.03227871774903718
mean pearson: -0.03516779866815263
mean cosine similarity: 0.2717763197530616


model: genie_standardscaler_tf
mean kendall-tau: 0.2650302951072529
mean pearson: 0.20773368959136437
mean cosine similarity: 0.3414355053875321


model: genie
mean kendall-tau: -0.03380536151458099
mean pearson: -0.03689937231651398
mean cosine similarity: 0.27089218327561176


model: genie_tf
mean kendall-tau: 0.2653074503105995
mean pearson: 0.20934239655164968
mean cosine similarity: 0.34267824221098403


model: genie_obs
mean kendall-tau: 0.03953201285874105
mean pearson: 0.07594009120189626
mean cosine similarity: 0.3275837701364394


model: genie_obs_tf
mean kendall-tau: 0.2690944794839005
mean pearson: 0.22108250724372203
mean cosine similarity: 0.35093538806884295

ngenes_20_interval_50_stepsize_0.0002

model: random
mean kendall-tau: -0.01915247940617325
mean pearson: -0.02632861878851484
mean cosine similarity: -0.03554611841537868


model: genie_standardscaler
mean kendall-tau: -0.04922081009844227
mean pearson: -0.043445920597908944
mean cosine similarity: 0.26371676905388175


model: genie_standardscaler_tf
mean kendall-tau: 0.27053860529648227
mean pearson: 0.2053320938614497
mean cosine similarity: 0.3334565820451548


model: genie
mean kendall-tau: -0.05031656490917482
mean pearson: -0.04443952373725054
mean cosine similarity: 0.26188496089665436


model: genie_tf
mean kendall-tau: 0.2700100591164239
mean pearson: 0.20528003592171318
mean cosine similarity: 0.3336270560187345


model: genie_obs
mean kendall-tau: 0.03437122564886416
mean pearson: 0.08511472598902388
mean cosine similarity: 0.33554127061836597


model: genie_obs_tf
mean kendall-tau: 0.2709385428943717
mean pearson: 0.22258528404558542
mean cosine similarity: 0.3483572903208703

ngenes_20_interval_100_stepsize_0.0001

model: random
mean kendall-tau: -0.025211057141589173
mean pearson: -0.03668595865580344
mean cosine similarity: -0.0451839004883816


model: genie_standardscaler
mean kendall-tau: -0.02133434348819712
mean pearson: -3.920517890667341e-05
mean cosine similarity: 0.29103592439617515


model: genie_standardscaler_tf
mean kendall-tau: 0.23037898617974553
mean pearson: 0.13208816711984459
mean cosine similarity: 0.2766695085561542


model: genie
mean kendall-tau: -0.021211193280340827
mean pearson: 0.002692495150877688
mean cosine similarity: 0.29216901934229883


model: genie_tf
mean kendall-tau: 0.23169766326045727
mean pearson: 0.13540421909767347
mean cosine similarity: 0.2792596018205943


model: genie_obs
mean kendall-tau: -0.006240872316302012
mean pearson: 0.030658906054633644
mean cosine similarity: 0.3005732626319872


model: genie_obs_tf
mean kendall-tau: 0.23075226524314385
mean pearson: 0.1336238958852612
mean cosine similarity: 0.28067492444273134

ngenes_20_interval_100_stepsize_0.0002

model: random
mean kendall-tau: -0.025211057141589173
mean pearson: -0.03668595865580344
mean cosine similarity: -0.0451839004883816


model: genie_standardscaler
mean kendall-tau: -0.04228426317181551
mean pearson: -0.061785357793663905
mean cosine similarity: 0.2552514493672183


model: genie_standardscaler_tf
mean kendall-tau: 0.22561318856885446
mean pearson: 0.12253268145211232
mean cosine similarity: 0.27187864015072266


model: genie
mean kendall-tau: -0.0419030358890254
mean pearson: -0.06138115661322301
mean cosine similarity: 0.2549956234604921


model: genie_tf
mean kendall-tau: 0.225619158957212
mean pearson: 0.12024011167626858
mean cosine similarity: 0.27001725566397045


model: genie_obs
mean kendall-tau: -0.010292211326394038
mean pearson: 0.0032867243426109496
mean cosine similarity: 0.2928023710215133


model: genie_obs_tf
mean kendall-tau: 0.22815657400917203
mean pearson: 0.1225187251052021
mean cosine similarity: 0.27131987156568743

ngenes_20_interval_200_stepsize_0.0001

model: random
mean kendall-tau: -0.03164584645799102
mean pearson: -0.04092940868579848
mean cosine similarity: -0.04909878692053782


model: genie_standardscaler
mean kendall-tau: -0.02429036907172031
mean pearson: -0.01509346634414635
mean cosine similarity: 0.29008049633874167


model: genie_standardscaler_tf
mean kendall-tau: 0.2372370155368052
mean pearson: 0.17765517635175995
mean cosine similarity: 0.31477809349318747


model: genie
mean kendall-tau: -0.02990379780851407
mean pearson: -0.01894875041463226
mean cosine similarity: 0.28689955120179594


model: genie_tf
mean kendall-tau: 0.23656781287655426
mean pearson: 0.1769470672446569
mean cosine similarity: 0.31478107012780815


model: genie_obs
mean kendall-tau: 0.06366192794214787
mean pearson: 0.1267750852076265
mean cosine similarity: 0.36427018127099653


model: genie_obs_tf
mean kendall-tau: 0.246862968852847
mean pearson: 0.2279948813672252
mean cosine similarity: 0.3586921856827564

ngenes_20_interval_200_stepsize_0.0002

model: random
mean kendall-tau: -0.03164584645799102
mean pearson: -0.04092940868579848
mean cosine similarity: -0.04909878692053782


model: genie_standardscaler
mean kendall-tau: -0.017756005583826357
mean pearson: -0.014728699977867831
mean cosine similarity: 0.2915069556819758


model: genie_standardscaler_tf
mean kendall-tau: 0.23997236194192328
mean pearson: 0.18097330335373113
mean cosine similarity: 0.3159990194012499


model: genie
mean kendall-tau: -0.01815859909120355
mean pearson: -0.0190087817991906
mean cosine similarity: 0.28836797141773024


model: genie_tf
mean kendall-tau: 0.23849650789814641
mean pearson: 0.17971841383493772
mean cosine similarity: 0.31495915234921346


model: genie_obs
mean kendall-tau: 0.0946342893369997
mean pearson: 0.12618824179185248
mean cosine similarity: 0.36125514389840224


model: genie_obs_tf
mean kendall-tau: 0.25455633341575906
mean pearson: 0.229237968391221
mean cosine similarity: 0.35637363496919683

ngenes_30_interval_50_stepsize_0.0001

model: random
mean kendall-tau: 0.04462301264970386
mean pearson: 0.05829746141107081
mean cosine similarity: 0.035160351198054295


model: genie_standardscaler
mean kendall-tau: -0.06374465654457281
mean pearson: -0.07565198439211475
mean cosine similarity: 0.1804254167208734


model: genie_standardscaler_tf
mean kendall-tau: 0.29480777073504055
mean pearson: 0.2063270653262489
mean cosine similarity: 0.2886237465068217


model: genie
mean kendall-tau: -0.06450815635941867
mean pearson: -0.0761345600861984
mean cosine similarity: 0.179763607152733


model: genie_tf
mean kendall-tau: 0.2945559264129942
mean pearson: 0.20726373047317556
mean cosine similarity: 0.28952756732288426


model: genie_obs
mean kendall-tau: 0.005972319337916277
mean pearson: 0.01762952747727031
mean cosine similarity: 0.20794503542289142


model: genie_obs_tf
mean kendall-tau: 0.29266484375476726
mean pearson: 0.19097347969853207
mean cosine similarity: 0.273751111420413

ngenes_30_interval_50_stepsize_0.0002

model: random
mean kendall-tau: 0.04462301264970386
mean pearson: 0.05829746141107081
mean cosine similarity: 0.035160351198054295


model: genie_standardscaler
mean kendall-tau: -0.05657258292853035
mean pearson: -0.06080965066819465
mean cosine similarity: 0.20556291529521617


model: genie_standardscaler_tf
mean kendall-tau: 0.29053344883736454
mean pearson: 0.20332327086474108
mean cosine similarity: 0.2866982513911181


model: genie
mean kendall-tau: -0.05639686616595688
mean pearson: -0.054751797894225324
mean cosine similarity: 0.20816855026970854


model: genie_tf
mean kendall-tau: 0.29119034116960973
mean pearson: 0.20287068958609047
mean cosine similarity: 0.2860179403428908


model: genie_obs
mean kendall-tau: 0.0102741466265172
mean pearson: 0.02289186222045552
mean cosine similarity: 0.21420987316862794


model: genie_obs_tf
mean kendall-tau: 0.2953150030287467
mean pearson: 0.21038823816314225
mean cosine similarity: 0.2919383350133029

ngenes_30_interval_100_stepsize_0.0001

model: random
mean kendall-tau: 0.04582962506121003
mean pearson: 0.056102674175451325
mean cosine similarity: 0.03284568924521926


model: genie_standardscaler
mean kendall-tau: -0.010830825029835707
mean pearson: 0.008934073524702438
mean cosine similarity: 0.21987172471440972


model: genie_standardscaler_tf
mean kendall-tau: 0.2920987539582827
mean pearson: 0.2105147285941132
mean cosine similarity: 0.2913436822506305


model: genie
mean kendall-tau: -0.010726483439796174
mean pearson: 0.011992470854385972
mean cosine similarity: 0.22191160810172716


model: genie_tf
mean kendall-tau: 0.292186175182705
mean pearson: 0.2097979480145218
mean cosine similarity: 0.2904932078997326


model: genie_obs
mean kendall-tau: -0.03287065699354561
mean pearson: -0.047489455821359786
mean cosine similarity: 0.1588526790273983


model: genie_obs_tf
mean kendall-tau: 0.2877167183601397
mean pearson: 0.1708478649425715
mean cosine similarity: 0.2531229252335656

ngenes_30_interval_100_stepsize_0.0002

model: random
mean kendall-tau: 0.04582962506121003
mean pearson: 0.056102674175451325
mean cosine similarity: 0.03284568924521926


model: genie_standardscaler
mean kendall-tau: -0.03312695626638039
mean pearson: -0.03173355228868729
mean cosine similarity: 0.19663218032965774


model: genie_standardscaler_tf
mean kendall-tau: 0.29080229823842496
mean pearson: 0.19753287691584842
mean cosine similarity: 0.2798319423835004


model: genie
mean kendall-tau: -0.029428086327976354
mean pearson: -0.027017881316837515
mean cosine similarity: 0.19915320766753852


model: genie_tf
mean kendall-tau: 0.2898233438130634
mean pearson: 0.19217335800261445
mean cosine similarity: 0.2748130707291971


model: genie_obs
mean kendall-tau: 0.0070812202434051945
mean pearson: -0.009331807898014534
mean cosine similarity: 0.18538159274282082


model: genie_obs_tf
mean kendall-tau: 0.2899469392341162
mean pearson: 0.17048866219073572
mean cosine similarity: 0.2519583043601974

ngenes_30_interval_200_stepsize_0.0001

model: random
mean kendall-tau: 0.055926890899859645
mean pearson: 0.05913245874987686
mean cosine similarity: 0.035417011503149666


model: genie_standardscaler
mean kendall-tau: 0.01976472856431633
mean pearson: 0.025479064875537527
mean cosine similarity: 0.24452202993866534


model: genie_standardscaler_tf
mean kendall-tau: 0.28996069009205794
mean pearson: 0.27312689808268853
mean cosine similarity: 0.3506831957653762


model: genie
mean kendall-tau: 0.017479006245496847
mean pearson: 0.01747109522953238
mean cosine similarity: 0.23846985627861272


model: genie_tf
mean kendall-tau: 0.2902390237620157
mean pearson: 0.2762149494704447
mean cosine similarity: 0.3535732249849337


model: genie_obs
mean kendall-tau: -0.03923597986599043
mean pearson: -0.05895530697481225
mean cosine similarity: 0.1443528023756199


model: genie_obs_tf
mean kendall-tau: 0.26926381575532404
mean pearson: 0.11101561602733366
mean cosine similarity: 0.2021428419404615

ngenes_30_interval_200_stepsize_0.0002

model: random
mean kendall-tau: 0.055926890899859645
mean pearson: 0.05913245874987686
mean cosine similarity: 0.035417011503149666


model: genie_standardscaler
mean kendall-tau: -0.002537093095632189
mean pearson: -0.010816306211116627
mean cosine similarity: 0.22302639263787694


model: genie_standardscaler_tf
mean kendall-tau: 0.2846623281411173
mean pearson: 0.2639984614322083
mean cosine similarity: 0.34178990288421596


model: genie
mean kendall-tau: -0.003403940759493371
mean pearson: -0.013814984640175417
mean cosine similarity: 0.22144763998715428


model: genie_tf
mean kendall-tau: 0.2839194605476532
mean pearson: 0.26064896126193565
mean cosine similarity: 0.33880854846743363


model: genie_obs
mean kendall-tau: -0.041481336702222975
mean pearson: -0.03825059721095901
mean cosine similarity: 0.15948030622581036


model: genie_obs_tf
mean kendall-tau: 0.27364392123231257
mean pearson: 0.12034869974225565
mean cosine similarity: 0.2097872766242504

ngenes_50_interval_50_stepsize_0.0001

model: random
mean kendall-tau: 0.004368558803362855
mean pearson: 0.014275850774625464
mean cosine similarity: 0.009820366420494334


model: genie_standardscaler
mean kendall-tau: 0.00788717257905631
mean pearson: 0.014260086211877314
mean cosine similarity: 0.15379787216340798


model: genie_standardscaler_tf
mean kendall-tau: 0.30953304550421845
mean pearson: 0.2095108809460073
mean cosine similarity: 0.25627640317383804


model: genie
mean kendall-tau: 0.008473410909201118
mean pearson: 0.01234842279113377
mean cosine similarity: 0.15279290022756062


model: genie_tf
mean kendall-tau: 0.309735276627565
mean pearson: 0.21214534808591617
mean cosine similarity: 0.2588049627263105


model: genie_obs
mean kendall-tau: -0.0008483177383119375
mean pearson: -0.009485104011364844
mean cosine similarity: 0.12375599127189427


model: genie_obs_tf
mean kendall-tau: 0.31195874119827444
mean pearson: 0.22167095237441595
mean cosine similarity: 0.26663118516032525

ngenes_50_interval_50_stepsize_0.0002

model: random
mean kendall-tau: 0.004368558803362855
mean pearson: 0.014275850774625464
mean cosine similarity: 0.009820366420494334


model: genie_standardscaler
mean kendall-tau: 0.018202948819873874
mean pearson: 0.012524517164558166
mean cosine similarity: 0.1541539982756891


model: genie_standardscaler_tf
mean kendall-tau: 0.30967167294795706
mean pearson: 0.21247617178904005
mean cosine similarity: 0.2596224560153096


model: genie
mean kendall-tau: 0.019687364109255576
mean pearson: 0.013804473944067048
mean cosine similarity: 0.15503863152299618


model: genie_tf
mean kendall-tau: 0.30983618058555007
mean pearson: 0.2131612780557069
mean cosine similarity: 0.26026552499493166


model: genie_obs
mean kendall-tau: -0.010297215122535395
mean pearson: -0.025612092409998757
mean cosine similarity: 0.11084711272985218


model: genie_obs_tf
mean kendall-tau: 0.30985927380027034
mean pearson: 0.19313216428073376
mean cosine similarity: 0.23953478668703018

ngenes_50_interval_100_stepsize_0.0001

model: random
mean kendall-tau: 0.0018929856948360182
mean pearson: 0.015810480965473953
mean cosine similarity: 0.011278591378557834


model: genie_standardscaler
mean kendall-tau: 0.01823119560154213
mean pearson: 0.0253382890453948
mean cosine similarity: 0.1643192728541773


model: genie_standardscaler_tf
mean kendall-tau: 0.3062806405887766
mean pearson: 0.25056316269030093
mean cosine similarity: 0.29489642009092315


model: genie
mean kendall-tau: 0.015878668213359044
mean pearson: 0.02075331799410779
mean cosine similarity: 0.16021887067427962


model: genie_tf
mean kendall-tau: 0.3062171236369684
mean pearson: 0.24962704085784357
mean cosine similarity: 0.2940222975460836


model: genie_obs
mean kendall-tau: 0.038491964725699115
mean pearson: 0.07682570153982887
mean cosine similarity: 0.18866379696889168


model: genie_obs_tf
mean kendall-tau: 0.3133742596286246
mean pearson: 0.3196164757661589
mean cosine similarity: 0.3602559987898236

ngenes_50_interval_100_stepsize_0.0002

model: random
mean kendall-tau: 0.0018929856948360182
mean pearson: 0.015810480965473953
mean cosine similarity: 0.011278591378557834


model: genie_standardscaler
mean kendall-tau: 0.0038262756685516945
mean pearson: -0.0017913779392503576
mean cosine similarity: 0.1483606173788002


model: genie_standardscaler_tf
mean kendall-tau: 0.30001773119646663
mean pearson: 0.21565390665010198
mean cosine similarity: 0.2622125908939429


model: genie
mean kendall-tau: 0.00046388207241147795
mean pearson: -0.008421981702618936
mean cosine similarity: 0.1435228644949993


model: genie_tf
mean kendall-tau: 0.30060599078976796
mean pearson: 0.21805144956261507
mean cosine similarity: 0.2644679696863247


model: genie_obs
mean kendall-tau: 0.052183412159615784
mean pearson: 0.09699221177030895
mean cosine similarity: 0.20597276776122866


model: genie_obs_tf
mean kendall-tau: 0.3137358610709522
mean pearson: 0.3135760307555296
mean cosine similarity: 0.3541067801181588

ngenes_50_interval_200_stepsize_0.0001

model: random
mean kendall-tau: 0.0056453514996287315
mean pearson: 0.022012152773166982
mean cosine similarity: 0.01731703734561031


model: genie_standardscaler
mean kendall-tau: 0.01328272043199931
mean pearson: 0.020629037177854328
mean cosine similarity: 0.16140443551677844


model: genie_standardscaler_tf
mean kendall-tau: 0.30340365678303466
mean pearson: 0.24468628498738826
mean cosine similarity: 0.29026719669621703


model: genie
mean kendall-tau: 0.014628708401440082
mean pearson: 0.02600524695834636
mean cosine similarity: 0.16505800255942568


model: genie_tf
mean kendall-tau: 0.3030501707487367
mean pearson: 0.24406526208068577
mean cosine similarity: 0.28971896792918256


model: genie_obs
mean kendall-tau: 0.03461925326908378
mean pearson: 0.0687956954408224
mean cosine similarity: 0.18323960859076108


model: genie_obs_tf
mean kendall-tau: 0.30778731177224095
mean pearson: 0.2888471640806149
mean cosine similarity: 0.33166833332174805

ngenes_50_interval_200_stepsize_0.0002

model: random
mean kendall-tau: 0.0056453514996287315
mean pearson: 0.022012152773166982
mean cosine similarity: 0.01731703734561031


model: genie_standardscaler
mean kendall-tau: 0.006625451587341564
mean pearson: -0.005458402531561188
mean cosine similarity: 0.14521462860686168


model: genie_standardscaler_tf
mean kendall-tau: 0.29803358559244997
mean pearson: 0.2131810539362661
mean cosine similarity: 0.2604026134688645


model: genie
mean kendall-tau: 0.00629875314972689
mean pearson: -0.010149735875215622
mean cosine similarity: 0.14195262074107212


model: genie_tf
mean kendall-tau: 0.29808309316439235
mean pearson: 0.21228779950399362
mean cosine similarity: 0.25955988855237355


model: genie_obs
mean kendall-tau: 0.0558206991222487
mean pearson: 0.12886249920436438
mean cosine similarity: 0.23083132968547665


model: genie_obs_tf
mean kendall-tau: 0.308865281599967
mean pearson: 0.2917975456881186
mean cosine similarity: 0.33395213385497724

ngenes_20_interval_50_stepsize_0.0001
ngenes_20_interval_50_stepsize_0.0002
ngenes_20_interval_100_stepsize_0.0001
ngenes_20_interval_100_stepsize_0.0002
ngenes_20_interval_200_stepsize_0.0001
ngenes_20_interval_200_stepsize_0.0002
ngenes_30_interval_50_stepsize_0.0001
ngenes_30_interval_50_stepsize_0.0002
ngenes_30_interval_100_stepsize_0.0001
ngenes_30_interval_100_stepsize_0.0002
ngenes_30_interval_200_stepsize_0.0001
ngenes_30_interval_200_stepsize_0.0002
ngenes_50_interval_50_stepsize_0.0001
ngenes_50_interval_50_stepsize_0.0002
ngenes_50_interval_100_stepsize_0.0001
ngenes_50_interval_100_stepsize_0.0002
ngenes_50_interval_200_stepsize_0.0001
ngenes_50_interval_200_stepsize_0.0002
genie_standardscaler
genie_standardscaler_tf
ngenes_20_interval_50_stepsize_0.0001
ngenes_20_interval_50_stepsize_0.0002
ngenes_20_interval_100_stepsize_0.0001
ngenes_20_interval_100_stepsize_0.0002
ngenes_20_interval_200_stepsize_0.0001
ngenes_20_interval_200_stepsize_0.0002
ngenes_30_interval_50_stepsize_0.0001
ngenes_30_interval_50_stepsize_0.0002
ngenes_30_interval_100_stepsize_0.0001
ngenes_30_interval_100_stepsize_0.0002
ngenes_30_interval_200_stepsize_0.0001
ngenes_30_interval_200_stepsize_0.0002
ngenes_50_interval_50_stepsize_0.0001
ngenes_50_interval_50_stepsize_0.0002
ngenes_50_interval_100_stepsize_0.0001
ngenes_50_interval_100_stepsize_0.0002
ngenes_50_interval_200_stepsize_0.0001
ngenes_50_interval_200_stepsize_0.0002
